packages/create/src/index.ts

```diff
@@ -0,0 +1,135 @@
+import fs from 'fs-extra';
+import path from 'path';
+import meow from 'meow';
+import enquirer from 'enquirer';
+import execa from 'execa';
+import ora from 'ora';
+import c from 'chalk';
+import terminalLink from 'terminal-link';
+import { checkVersion } from './checkVersion';
+import { UserError } from './utils';
+import { fileURLToPath } from 'url';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
```

Comment:
Marked the package with `"type": "module"` in the `package.json` so we cannot use `__dirname` (not available in ESM) had to use [this alternative](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import.meta#resolving_a_file_relative_to_the_current_one)

PS: The latest version of Node introduced back [dirname for ESM via import.meta ](https://nodejs.org/api/esm.html#importmeta)